api/envoy/extensions/filters/http/oauth2/v3/oauth.proto

```diff
@@ -70,6 +70,10 @@ message OAuth2Credentials {
 
   // The cookie names used in OAuth filters flow.
   CookieNames cookie_names = 4;
+
+  // The domain to set the cookie on. If not set, the cookie will be set on the domain of the request.
```

Comment:
If the unset behavior the same as existing behavior?

Reply:
Yes, if unset the behavior doesn't change, defaulting to the host of the request, not including the subdomains.